pubsub/cloud-client/subscriber.py

```diff
@@ -90,32 +105,41 @@ def create_push_subscription(project,
     # [END pubsub_create_push_subscription]
 
 
-def delete_subscription(project, subscription_name):
+def delete_subscription(project_id, subscription_name):
     """Deletes an existing Pub/Sub topic."""
     # [START pubsub_delete_subscription]
-    # project           = "Your Google Cloud Project ID"
-    # subscription_name = "Your Pubsub subscription name"
+    from google.cloud import pubsub_v1
+
+    # TODO project_id = "Your Google Cloud Project ID"
+    # TODO subscription_name = "Your Pub/Sub subscription name"
+
     subscriber = pubsub_v1.SubscriberClient()
     subscription_path = subscriber.subscription_path(
-        project, subscription_name)
+        project_id, subscription_name)
 
     subscriber.delete_subscription(subscription_path)
 
     print('Subscription deleted: {}'.format(subscription_path))
     # [END pubsub_delete_subscription]
 
 
-def update_subscription(project, subscription_name, endpoint):
+def update_subscription(project_id, subscription_name, endpoint):
     """
     Updates an existing Pub/Sub subscription's push endpoint URL.
     Note that certain properties of a subscription, such as
     its topic, are not modifiable.
     """
     # [START pubsub_update_push_configuration]
-    # endpoint = "https://my-test-project.appspot.com/push"
+    from google.cloud import pubsub_v1
+
+    # TODO project_id = "Your Google Cloud Project ID"
+    # TODO topic_name = "Your Pub/Sub topic name"
+    # TODO subscription_name = "Your Pub/Sub subscription name"
+    # TODO endpoint = (for example, "https://my-test-project.appspot.com/push")
```

Comment:
Why "for example" here and not elsewhere? Seems obvious to me that these are all examples, so I'd drop it.